accord-core/src/test/java/accord/utils/RandomSourceTest.java

```diff
@@ -19,25 +19,22 @@
 package accord.utils;
 
 import java.util.Arrays;
-import java.util.Random;
 
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static accord.utils.RandomTestRunner.test;
+
 public class RandomSourceTest
 {
     private static final Logger logger = LoggerFactory.getLogger(RandomSourceTest.class);
 
     @Test
     public void testBiasedInts()
     {
-        RandomSource random = RandomSource.wrap(new Random());
-        long seed = random.nextLong();
-        logger.info("Seed: {}", seed);
-        random.setSeed(seed);
-        testBiasedInts(random, 1000, 100000, 0.01, 0.1);
+        test().check(random -> testBiasedInts(random, 1000, 100000, 0.01, 0.1));
```

Comment:
this is also in https://issues.apache.org/jira/browse/CASSANDRA-19114, its a small change so hope that can go in quickly... I just want stable CI so had to fix this flakey test